apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -174,13 +179,15 @@ class MoveStudents extends Component {
         header: {
           label: i18n.name(),
           props: {
+            id: 'name-header',
```

Comment:
nit: when these are for testing only, we typically prepend with `uitest` like this: https://github.com/code-dot-org/code-dot-org/blob/9ea4132bda304653fb4dccf29a90a993ca3d3ce9/apps/src/templates/studioHomepages/SetUpSections.jsx#L22